mail_tracking/models/mail_message.py

```diff
@@ -29,7 +30,7 @@ class MailMessage(models.Model):
     )
     is_failed_message = fields.Boolean(
         compute="_compute_is_failed_message",
-        search="_search_is_failed_message",
+        store=True,
```

Comment:
Do we really need to make it stored?

Reply:
Hi, this flag is used to populate the controller: `/mail/failed/messages`

Reply:
I just tested on runbot and still see demo data, maybe another try?

![2024-10-18_12-36](https://github.com/user-attachments/assets/86593755-dedd-4396-9b24-7140cda70975)

Reply:
Please don't change how it was, no stored and with search method. Or does any reason for changing it?

Reply:
Hi @pedrobaeza, yes I agree there is no need to store it in db. But look like the search method is not working. 

I am working on it, will update when it's done

Reply:
Hi @pedrobaeza @chienandalu , I just added [fixup! [MIG] mail_tracking: Migration to 18.0](https://github.com/OCA/social/pull/1476/commits/a6bce24c379bb8c8b3be606470b24decfc0fd427)

Reply:
Hi @pedrobaeza @chienandalu , I just added [fixup! [MIG] mail_tracking: Migration to 18.0](https://github.com/OCA/social/pull/1476/commits/a6bce24c379bb8c8b3be606470b24decfc0fd427)

- As I inspected, _search_is_failed_message was not working. Without this commit, the condition in final query is always False => return no result
- I added a test to make sure it's covered